data/replaced_tags.txt

```diff
@@ -172,3 +172,6 @@ usage=freight : usage=utility
 service=siding : service=service
 service=yard : service=service
 service=crossover : service=service
+
+denomination=latter_day_saints : denomination=mormon
```

Comment:
what about other options mentioned in https://github.com/organicmaps/organicmaps/issues/4931#issuecomment-1507196692
do they need to be included as well?

Reply:
I'm not sure but I think denomination=la_luz_del_mundo and denomination=iglesia_ni_cristo churches are different to LDS or mormon and may need their own name string and icon? I'm not really sure